source/mainnet/net/guides/run-local-chain.rst

```diff
@@ -0,0 +1,185 @@
+
+Running the tool
+----------------
+
+The ``genesis-creator`` tool uses a TOML configuration file format for specifying parameters from which the genesis data is generated. The TOML file specifies
+
+* the initial protocol version
+* cryptographic parameters
+* anonymity revokers
+* identity providers
+* foundation accounts
+* keys for updating the chain
+* various parameters for the genesis
+
+Furthermore, it specifies where to save the output that is used to invoke the node binary. Many of these options are not relevant when testing smart contracts and the easiest way to get started is to piggyback on one of the examples in the ``./examples`` folder. In the following, you will use the file ``./examples/genesis5.toml`` and modify it slightly. Inspecting the file reveals that it specifies an initial protocol version of 5, to output credentials for 5 baker accounts, 2 foundation accounts, and the genesis time set to the system time at generation. It also specifies 5 seconds as the average time per block. Further inspection of the table at the ``accounts`` key of the file reveals that the bakers each have an initial balance of 10^15 microCCD and a stake of 5 * 10^14 microCCD. Change this section such that only one baker credential is produced by setting the value of the ``repeat`` key to 1 as follows:
+
+.. code-block:: toml
+
+    [[accounts]]
+    kind = "fresh"
+    balance = "1000000000000000"
+    stake = "500000000000000"
+    template = "baker"
+    identityProvider = 0
+    numKeys = 1
+    threshold = 1
+    repeat = 1 # Changed from 5
+
+.. Note::
+
+    Note that the staked amount needed to participate in the finalization committee is some fraction of the total amount of existing CCD defined by the value of the ``capitalBound`` key in the configurations file. The total amount is the sum of the balances of all the baker and foundation accounts specified in the genesis configuration file. In this particular example, the stake is sufficient for baking.
+
+Next, generate the genesis data:
+
+.. code-block:: console
+
+    $ ./target/release/genesis-creator generate --config ./examples/genesis5.toml
+    Deleting any existing directories.
+    Account keys will be generated in ./accounts
+    Chain update keys will be generated in ./update-keys
+    Identity providers will be generated in ./idps
+    Anonymity revokers will be generated in ./ars
+    Baker keys will be generated in ./bakers
+    Cryptographic parameter will be generated in ./global
+    The genesis data will be stored in ./genesis.dat
+    The genesis hash will be written to ./genesis_hash
+    There are 2 accounts in genesis, 1 of which are bakers.
+    Genesis time is set to 2023-05-22 15:08:19.803 UTC.
+    Average block time is set to 5000ms.
+    DONE
+
+In particular, the files of interest to you are ``./genesis.dat``, containing the genesis block data, and ``./bakers/baker-0-credentials.json``, containing the credentials of the single baker account that was created.
```

Comment:
I think the files in `accounts` are also of interest, and can be imported into concordium-client.